### PR TITLE
Awesome books: a complete website with navigation

### PR DESCRIPTION
**Pull request by maov19**
Following features were added:

- Added navigation bar.
- Added three content sections:
> Books list.
> Add book form.
> Contact info.
- When clicked, only selected areas are displayed.

### CODE REVIEW

**Comment by DammyShittu** [comment]
Hi @maov19 and @App-Developer8866,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### Update your readme using the template. Ensure all headers are present

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by maov19** [comment]
Hi!
Readme was updated with the adequate template.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,155 @@
...
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (nahnahdev) [inline README.md:153]: - [ OPTIONAL ] The link to your MIT license leads to an error page. Kindly ensure that the link in the README file leads to the license you added to your project and not to an error page. :100:

File: style.css [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+  border-radius: 5px;
+}
+
+.element {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+
+  /* margin-bottom: 10px; */
```
Comment (nahnahdev) [inline style.css:66]: - [ OPTIONAL ] Kindly avoid leaving commented pieced of code in your codebase. It makes your code unclean :+1:

**Review by nahnahdev — APPROVED** [review]
Hi @App-Developer8866  and @maov19 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- [x] All project requirements completed :heavy_check_mark: 
- [x] Good use of GitHub flow :heavy_check_mark: 
- [ ] No linter errors :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (maov19) [reply README.md:153]: fixed, thanks!

File: style.css (hunk 2 above)
Comment (maov19) [reply style.css:66]: fixed, thanks!

**Pull request merged**